rlclientlib/live_model_impl.cc

```diff
@@ -91,8 +91,8 @@ namespace reinforcement_learning {
 
   int live_model_impl::request_decision(const char* context_json, unsigned int flags, decision_response& resp, api_status* status)
   {
-    if (_learning_mode == IMITATION) {
-      // Imitaion mode is not supported here at this moment
+    if (_learning_mode == APPRENTICE) {
```

Comment:
Is logging only supported here?

Reply:
good catch. fixed as loggingonly is not supported either.

Reply:
the idea for loggingonly mode is that we basically ignore the result from choose_rank like what we did for apprentice. So we return a response as actions in the same order as the input. Also adding a flag for other stuffs to be handled in joiner/trainer. Please let me know if you have any concerns and/or other questions.